src/test/java/com/epam/page/object/generator/integrationalTests/MainTest.java

```diff
@@ -172,4 +172,63 @@ public void pageObjectsGenerator_GenerateDropdownElementWithInnerElements() thro
         pog.generatePageObjects();
     }
 
+    @Test(expected = ValidationException.class)
+    public void pageObjectGenerator_notExistenceRootTitle() throws Exception {
+        PageObjectsGenerator pog = initPog(
+            "src/test/resources/dropdown-without-root.json",
+            "http://materializecss.com/dropdown.html",
+            false,
+            false);
+
+        pog.generatePageObjects();
```

Comment:
I would like to see exception message asserted as well in this integration testset.

Reply:
About this integration testset I can say that it is no longer relevant, a new part of team works with it and they provide a new integration testset as soon as possible.